src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.CallsMicrosoftGraph.razor

```diff
@@ -1,5 +1,5 @@
 ﻿<div class="top-row pl-4 navbar navbar-dark">
-    <a class="navbar-brand" href="">BlazorServerWeb-CSharp</a>
+    <a class="navbar navbar-brand" href="">BlazorServerWeb-CSharp</a>
```

Comment:
I think this should just be `class="navbar-brand"`

Reply:
Since this is the most widespread change, I'll wait til someone else confirms that we don't need this before undoing that across the board

Reply:
Undone